grama/eval_pnd.py

```diff
@@ -121,21 +120,32 @@ def eval_pnd(model, df_train, df_test, sign, n=int(1e4), seed=None):
 
     ### Create covariance matrices
     X_cov = zeros((X_sig.shape[0], length, length))
-    for i in range(X_sig.shape[0]):
-        X_cov[i, 0, 0] = X_sig[i, 0]
-        X_cov[i, 1, 1] = X_sig[i, 1]
+    for var in range(length):
+        for i in range(X_sig.shape[0]):
+            X_cov[i, var, var] = X_sig[i, var]
 
     ### Apply pnd
     pr_scores, var_values = approx_pnd(
         X_pred,
         X_cov,
         X_train,
-        sign = sign,
+        signs = signs,
         n = n,
         seed = seed
     )
 
-    return pr_scores, var_values
+    ### Package outputs
+    # pnd_results = DataFrame(pr_scores) + DataFrame(var_values)
+    pnd_results = DataFrame(
+        {
+            "pr_scores": pr_scores,
+            "var_values": var_values,
+        }
+    )
+
+    if append:
+        return df_test.merge(pnd_results, left_index=True, right_index=True)
```

Comment:
`merge` is probably fine, but we could get into trouble if the user provides a `df_test` that has its `index` column in an odd order. Since we're constructing `pnd_results` (which, btw, should be named something like `df_res` or `df_pnd`), you could do something like

`df_test.reset_index(drop=True).merge(pnd_results, left_index=True, right_index=True)`

Do you need to import `merge()` if you're calling it as a DataFrame method?